argilla/docs/getting_started/quickstart.md

```diff
@@ -133,8 +163,9 @@ dataset.records.log(records=data, mapping={"text": "review"})
 🎉 You have successfully created your first dataset with Argilla. You can now access it in the Argilla UI and start annotating the records.
 
 ## Next steps
+
```

Comment:
We could have a next step to upload data to the hub, so that users could use this feature without code? Or maybe this should be higher up.